SU-SWS/stanford_paragraph_types
# EARTH-91: CTA Paragraph Features.

# READY FOR REVIEW/

# Summary
- Call to action paragraph types. 

# Needed By (Date)
- 2016.06.14

# Urgency
- Moderate

# Steps to Test

0. Refresh local environment (optional)
1. Check out this branch
2. Check out (https://github.com/SU-SWS/stanford_components/pull/4) if it hasn't already been merged. If it hasn't been merged go test that one first and merge it. This is dependant on that PR.
3.  Enable `drush en stanford_paragraph_cta stanford_paragraph_cta_list`
5. Navigate to `/node/add/complex_page`
6. If the two new paragraphs don't show up as an option, edit the complex page component field to include them. There is a PR already to fix this.
7. Create the CTA List type and fill in all of the fields
8. Save the node and view the display. It should look like the film-strip component. New theme to come later.

# Affected Projects or Products
- https://github.com/SU-SWS/stanford_components/pull/4

# Associated Issues and/or People
- EARTH-91
- https://github.com/SU-SWS/stanford_components/pull/4

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
